adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -123,6 +126,139 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] {
     replaceRdd(tFn(rdd))
   }
 
+  def sorted: Boolean
+  // The partition map is structured as follows:
+  // The outer option is for whether or not there is a partition map.
+  //   - This is None in the case that we don't know the bounds on each 
+  //     partition.
+  // The Array is the length of the number of partitions.
+  // The inner option is in case there is no data on a partition.
+  // The (ReferenceRegion, ReferenceRegion) tuple contains the bounds of the 
+  //   partition, such that the lowest start is first and the highest end is
+  //   second.
+  def partitionMap: Option[Array[Option[(ReferenceRegion, ReferenceRegion)]]]
```

Comment:
Yeah, but that access in `Mango` should be through `filterOverlappingReferenceRegions`, right?